pages/treeherder.py

```diff
@@ -86,6 +87,10 @@ def result_sets(self):
         return [self.ResultSet(self, el) for el in self.find_elements(*self._result_sets_locator)]
 
     @property
+    def results_visible(self):
+        return self.find_element(*self._resultset_visible_locator)
```

Comment:
Would it make sense to put this into `wait_for_page_to_load`? That way we'll be implicitly checking this by waiting for it to be true every time we instantiate a page. Also, we can add an explicit call anywhere a page load is expected. For example, put `self.wait_for_page_to_load()` into `get_next_ten_results` instead of adding asserts to the tests.